features/sctp/README.md

```diff
@@ -1,7 +1,92 @@
-# How to enable SCTP on OCP 4.3
+# SCTP on OpenShift
 
-## Load the sctp kernel module
+OpenShift Container Platform includes the capability of using (single homed) SCTP connections.
+The Stream Control Transmission Protocol (SCTP) is a computer networking communications protocol which operates at the transport layer and serves a role similar to the popular protocols TCP and UDP.
 
-The SCTP module is blacklisted by default, so in order to use it we need to unblacklist it and load it at boot time.
+Withing the OpenShift Container Platform you can:
 
-This is achieved by the Machine Config configuration file [sctp_module_mc.yaml](./sctp_module_mc.yaml).
+- establish single homed pod to pod sctp connections
+- expose SCTP ClusterIP Services
+- expose SCTP NodePort Services
+
+This is achieved by specifying the `protocol` field to `SCTP` the same way is done with `TCP`. 
+
+For example:
+
+```yaml
+apiVersion: v1
+kind: Service
+metadata:
+  name: demoserver
+  namespace: sctp-demo
+  labels:
+    app: demoserver
+spec:
+  selector:
+    app: demoserver
+  ports:
+    - name: demoserver
+      protocol: SCTP
+      port: 30101
+      targetPort: "demoserver"
+```
+
+## Enabling SCTP
+
+The `SCTP` protocol is enabled by default in OpenShift. Nothing prevents the creation of Pods exposing SCTP ports.
+
+However, in order to be able to use it some tweaking is necessary. This is because the SCTP kernel module is blacklisted by default.
+
+In order to have the applications able to load the module, two different actions need to be performed:
+
+- unblacklisting the kernel module
+- having it loaded at boot time
+
+This can be achieved applying a [machine configuration file](https://github.com/openshift/machine-config-operator) like the one below:
+
+``` yaml
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: worker
+  name: load-sctp-module
+spec:
+  config:
+    ignition:
+      version: 2.2.0
+    storage:
+      files:
+        - contents:
+            source: data:,
+            verification: {}
+          filesystem: root
+          mode: 420
+          path: /etc/modprobe.d/sctp-blacklist.conf
+        - contents:
+            source: data:text/plain;charset=utf-8;base64,c2N0cA==
```

Comment:
AFAIK, since 'sctp' is within the 'safe range', it'll work without base64
and be more readable as

`source: data:text/plain;charset=utf-8;sctp`

Reply:
Yes, I just tried it to be sure.
According to the [rfc](https://tools.ietf.org/html/rfc2397), it should be `data:text/plain;charset=utf-8,sctp` (and it worked this way).

Will update both the manifest & the doc.